### PR TITLE
Profile screen

### PR DESCRIPTION
**Pull request by lucasaarcoverde**
captura de tela 2018-06-30 as 19 45 12

### CODE REVIEW

File: frontend/src/components/Profile/Badge.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const bgColors = { 
+  "Blue": "#6495ED",
+  "Green": "#32CD32",
+  "Yellow": "#FFD700",
+};
+
+export default class Badge extends Component {
+  constructor(props) {
+    super(props);
+  }
```
Comment (raonismaneoto) [inline frontend/src/components/Profile/Badge.js]: insert semicolon

File: frontend/src/components/Profile/Badge.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+ 	render() {
```
Comment (raonismaneoto) [inline frontend/src/components/Profile/Badge.js]: is this indentation ok?

File: frontend/src/components/Profile/BadgesCard.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import React, {Component} from 'react';
+import Badge from './Badge';
+
+export default class BadgesCard extends Component {
+  constructor(props) {
+    super(props);
+  }
```
Comment (raonismaneoto) [inline frontend/src/components/Profile/BadgesCard.js]: missing semicolon

File: frontend/src/components/Profile/BadgesCard.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+	    	<p className="f3 blue ml3">Badges</p>
+	    	<div className="h6 ma2 w-98 ba b--dark-blue">
+		      <Badge bgColor="Blue" contentText="> 1000 on Codeforces"/>
+		      <Badge bgColor="Yellow" contentText="Dynamic Programming Master"/>
+		      <Badge bgColor="Green" contentText="Added a Theme"/>
+	    	</div>
+	    </article>
+	  );
+  }
+}
```
Comment (raonismaneoto) [inline frontend/src/components/Profile/BadgesCard.js]: missing semicolon

File: frontend/src/components/Profile/BadgesCard.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import React, {Component} from 'react';
+import Badge from './Badge';
+
+export default class BadgesCard extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+ 	render() {
```
Comment (raonismaneoto) [inline frontend/src/components/Profile/BadgesCard.js]: indentation

File: frontend/src/components/Profile/Profile.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import React, {Component} from 'react';
```
Comment (raonismaneoto) [inline frontend/src/components/Profile/Profile.js]: same thing to this component

File: frontend/src/components/Profile/Badge.js [hunk 7]
```diff
@@ -0,0 +1,22 @@
+import React, {Component} from 'react';
+
+const bgColors = { 
+  "Blue": "#6495ED",
```
Comment (gugabribeiro) [inline frontend/src/components/Profile/Badge.js]: Use tachyons colors.

File: frontend/src/components/Profile/Badge.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+
+export default class Badge extends Component {
+  constructor(props) {
+    super(props);
+  };
+
+ 	render() {
+ 		let {bgColor, contentText} = this.props;
```
Comment (gugabribeiro) [inline frontend/src/components/Profile/Badge.js]: Change this let statement for a const one.

File: frontend/src/components/Profile/Badge.js [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+ 		let badgeClass = `h-10 dib w5 ma3 br3 ba b--black-10 bg-${bgColor}`;
```
Comment (gugabribeiro) [inline frontend/src/components/Profile/Badge.js]: This too.

**Pull request merged**